src/App.vue

```diff
@@ -204,43 +394,134 @@ export default {
         // },
 
         favorites(favorites) {
-            localStorage.setItem('favorites', JSON.stringify(favorites))
+            localStorage.setItem('favorites', JSON.stringify(favorites));
 
-            return favorites
+            return favorites;
         },
     },
 
     methods: {
+        addColumn () {
+            this.inputColumn.push({id:(this.index++), name:'', type: ''});
+        },
+
+        removeRow (row) {
+            this.inputColumn.splice(row, 1);
+        },
+
+        toggleModal (title, add, type) {
+            this.showDialogCreate = !this.showDialogCreate;
+            this.dialogTitle = title;
+            this.showAddInput = add;
+            this.dataType = type;
+        },
+
+        closeModal () {
+            this.showDialogCreate = false;
+            this.newName.name = null;
+        },
+
+        capitalize (word) {
+            return word.charAt(0).toUpperCase() + word.slice(1);
+        },
+
+        getElementsColumns (inputColumn) {
+            let columnName=[], columnType=[], elements=[];
+
+            this.inputColumn.forEach(input => {
+                columnName.push( this.capitalize( (input.name).toLowerCase() ) );
+                columnType.push( (input.type).toUpperCase() );
+            })
+
+            for (let i = 0; i < columnName.length; i++) {
+                if (i != columnName.length -1) {
+                    elements += columnName[i] + " " + columnType[i] + ", ";
+                } else {
+                    elements += columnName[i] + " " + columnType[i];
+                }
+            }
+            return elements;
+        },
```

Comment:
You can refactor this a little bit. Just an idea using lodash. You don't need to receive the `inputColumn` as an argument

```js
return _
    .chain(this.inputColumn)
    .transform((acc, input, index) => {
        let name = _.capitalize(input.name.toLowerCase());
        let type = input.type.toUpperCase();

        acc[index] = `${name} ${type}`;
    })
    .join(',');
```